hw/opentitan/ot_hmac.c

```diff
@@ -762,12 +762,15 @@ static void ot_hmac_realize(DeviceState *dev, Error **errp)
 static void ot_hmac_reset(DeviceState *dev)
 {
     OtHMACState *s = OT_HMAC(dev);
+    OtHMACRegisters *r = s->regs;
 
     ibex_irq_set(&s->clkmgr, false);
 
     memset(s->ctx, 0, sizeof(*(s->ctx)));
     memset(s->regs, 0, sizeof(*(s->regs)));
 
+    r->cfg = 0x4100;
```

Comment:
Please favor an unsigned constant (`u` suffix)

Reply:
(or even better use the REG32 macro if you want to, this is something we failed to do up to now :-()